CRM/Admin/Form/Setting/Case.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/**
+ * This class generates form components for CiviCase.
+ */
+class CRM_Admin_Form_Setting_Case extends CRM_Admin_Form_Setting {
+
+  protected $_settings = array(
+    'civicaseRedactActivityEmail' => CRM_Core_BAO_Setting::SYSTEM_PREFERENCES_NAME,
+    'civicaseAllowMultipleClients' => CRM_Core_BAO_Setting::SYSTEM_PREFERENCES_NAME,
+    'civicaseNaturalActivityTypeSort' => CRM_Core_BAO_Setting::SYSTEM_PREFERENCES_NAME,
+    'civicaseActivityRevisions' => CRM_Core_BAO_Setting::SYSTEM_PREFERENCES_NAME,
+    'civicaseAllowCaseLocks' => CRM_Core_BAO_Setting::SYSTEM_PREFERENCES_NAME,
+  );
+
```

Comment:
Instead of overriding the  CRM_Admin_Form_Setting_Case, I experimented with a couple of hooks to see If we can do this with hooks.

We can set the add the `civicaseAllowCaseLocks` into the $_settings variable before the form is built.
```php
function hook_civicrm_preProcess($formName, &$form) {
  if ($formName == 'CRM_Admin_Form_Setting_Case') {
    $setting = $form->getVar('_settings');
    $setting['civicaseAllowCaseLocks'] = CRM_Core_BAO_Setting::SYSTEM_PREFERENCES_NAME;
    $form->setVar('_settings', $setting);
  }
}
```

For changing the Page/Form Title

```function hook_civicrm_alterMenu(&$items) {
$items['civicrm/admin/setting/case']['title'] = 'Settings - CiviCase';
}
```